### PR TITLE
General improvements

### PR DESCRIPTION
**Pull request by Tom94**

### CODE REVIEW

File: osu.Framework.Desktop/Platform/Windows/WindowsGameHost.cs [hunk 1]
```diff
@@ -16,12 +16,12 @@ namespace osu.Framework.Desktop.Platform.Windows
 {
     public class WindowsGameHost : DesktopGameHost
     {
-        public override bool IsActive => Window != null && GetForegroundWindow().Equals(Window.Handle);
-
         private TimePeriod timePeriod;
 
         internal WindowsGameHost(GraphicsContextFlags flags)
         {
+            IsActive = Window != null && GetForegroundWindow().Equals(Window.Handle);
```
Comment (peppy) [inline osu.Framework.Desktop/Platform/Windows/WindowsGameHost.cs]: what's going on here?

Comment (peppy) [inline osu.Framework.Desktop/Platform/Windows/WindowsGameHost.cs]: test

Comment (Tom94) [inline osu.Framework.Desktop/Platform/Windows/WindowsGameHost.cs]: The window might start in an inactive state, so we should make sure to set it to inactive in such a case.

Comment (peppy) [inline osu.Framework.Desktop/Platform/Windows/WindowsGameHost.cs]: Not sure I like the active logic being split across two different methods of checking whether active.. probably best to put a comment on this one just making this clear at very least (wasn't initially obvious to me).

**Pull request merged**